src/main/java/teammates/common/util/ActivityLogEntry.java

```diff
@@ -572,23 +574,33 @@ private String getStudentHomePageViewLink(String googleId) {
 
     public String getLogInfoForTableRowAsHtml() {
         return "<tr" + (isFirstRow ? " id=\"first-row\"" : "") + ">"
-               + "<td class=\"" + getTableCellColorCode(timeTaken) + "\" style=\"vertical-align: middle;\">"
-               + "<span><a onclick=\"submitLocalTimeAjaxRequest('" + time + "','" + googleId + "','" + role + "',this);\">" + getDateInfo() + "</a>"
-               + "<p class=\"localTime\"></p></span>"
-               + "<p class=\"" + getColorCode(getTimeTaken()) + "\">"
-               + "<strong>" + TimeHelper.convertToStandardDuration(getTimeTaken()) + "</strong>"
-               + "</p> </td> <td class=\"" + getTableCellColorCode(timeTaken) + "\">"
-               + "<form method=\"get\" action=\"" + Const.ActionURIs.ADMIN_ACTIVITY_LOG_PAGE + "\"> "
-               + "<h4 class=\"list-group-item-heading\">"
-               + getIconRoleForShow() + "&nbsp;" + getActionInfo() + "&nbsp;"
-               + "<small> id:" + id + " " + getPersonInfo() + "</span>" + "&nbsp;"
-               + "<button type=\"submit\" class=\"btn " + getLogEntryActionsButtonClass() + " btn-xs\">"
-               + "<span class=\"glyphicon glyphicon-zoom-in\"></span>"
-               + "</button> <input type=\"hidden\" name=\"filterQuery\" value=\"person:" + getAvailableIdenficationString() + "\">"
-               + "<input class=\"ifShowAll_for_person\" type=\"hidden\" name=\"all\" value=\"false\">"
-               + "<input class=\"ifShowTestData_for_person\" type=\"hidden\" name=\"testdata\" value=\"false\">"
-               + "</small> </h4> <div>" + getMessageInfo()
-               + "</div> </form> </td> </tr>";
+                + "<td class=\"" + getTableCellColorCode(timeTaken) + "\" style=\"vertical-align: middle;\">"
+                    + "<a onclick=\"submitLocalTimeAjaxRequest('" + time + "','" + googleId + "','" + role + "',this);\">"
+                            + getDateInfo() + "</a>"
```

Comment:
For future reference: 8 spaces being is used if it is extending a HTML tag, while 4 spaces is used for its contents eg.

```
String html = "<button id=\"clear-input-btn\" class=\"btn btn-default\" "
                    + "onclick=\"clearInput(this)\">"
                + "<span class=\"glyphicon glyphicon-remove\"></span>"
                + "Clear Input"
            + "</button>";
```